easybuild/easyconfigs/u/udocker/udocker-1.3.4-fosscuda-2020b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'udocker'
+version = '1.3.4'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
Please remove this (this is one of the things the CI is complaining about). Nowadays, we assume Python-3.X to be standard, so we don't add a version suffix for it anymore. We'll only add a versionsuffix if something still depends on Python-2.X.